opennms-doc/guide-admin/src/asciidoc/index.adoc

```diff
@@ -172,6 +172,12 @@ include::text/operation/newts/configuration.adoc[]
 include::text/operation/newts/cassandra-jmx.adoc[]
 include::text/operation/newts/cassandra-newts-jmx.adoc[]
 
+[[ga-opennms-operation-daemon-config-files]]
+=== Daemon Configuration Files
+include::text/operation/daemon-config-files/introduction.adoc[]
```

Comment:
Where are introduction.adoc and pollerd.adoc?  They're not in this PR, jus the eventd.adoc is.